packages/mwp-app-render/src/util/dateTimeUtils.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Pads numbers less than two digits with an additional
+ * leading zero. Used for date and time formatting.
+ * @param {Number} num  Number to be padded
+ * @returns {String}    Returns a padded string
+ */
+export const padNumber = num => (num < 10 ? `0${num}` : `${num}`);
```

Comment:
I'm wondering if we should create a story to change the `mup-web` codebase to use these functions from the platform and remove the duplicates there?

Reply:
Yeah, in fact I need to do a similar exercise with `getSocialLinks()` that's like in every repo we have now.